src/fixtures/data.jsx

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
Didn't spot this before but this file was merged in the previous PR so shouldn't be in this one as well?